process.rb

```diff
@@ -147,11 +147,18 @@ def slugify(word)
           .sort_by do |candidate|
             [-1 * (candidate.calculation(:total_contributions) || 0.0), candidate.Candidate]
           end
-          .map { |candidate| candidate.Candidate }
+
+      ContributionsByOrigin[election_name][:race_totals] ||= []
+      ContributionsByOrigin[election_name][:race_totals].append({
+        'title': office_election.title,
+        'type': 'office',
+        'slug': slugify(office_election.title),
+        'amount': candidates.sum {|candidate| candidate.calculation(:total_contributions) || 0.0}
+      })
```

Comment:
ruby style nitpick:

```ruby
ContributionsByOrigin[election_name][:race_totals].append(
  title: office_election.title,
  # ...
)
```

(keys shouldn't be enclosed in strings - they will be converted to symbols anyway unless you use the `=>` syntax like `"title" => "foo"`)